libmate-desktop/mate-bg.c

```diff
@@ -2973,7 +2973,7 @@ slideshow_unref (SlideShow *show)
 
 	g_queue_free (show->slides);
 
-	g_list_foreach (show->stack->head, (GFunc) g_free, NULL);
+	g_list_free_full (show->stack->head, g_free);
```

Comment:
Maybe even use `g_queue_free_full`, like in https://github.com/GNOME/gnome-desktop/commit/110f2700725195374a2d3ccfdf2e32f252b0accb.

Reply:
@monsta done. Thank you very much.